samza-shell/src/main/bash/run-class.sh

```diff
@@ -69,10 +69,7 @@ if [ -d "$JOB_LIB_DIR" ] && [ "$JOB_LIB_DIR" != "$BASE_LIB_DIR" ]; then
   echo generated combined CLASSPATH=$CLASSPATH
 else
   #default behavior
```

Comment:
Please add a comment and describe the behavior of the `.*` wildcard java argument( General notion of `dir/.*`  represents all the files in the directory(dir). Here it is different and it includes only .JAR files in the directory to class-path).

Reply:
1. Will test with a few cases, but really curious why do we need to add .war files since these are supposed to be not libraries and deployed using servlet/jsp
In my opinion, if there is some library level logic in a WAR file, a person should extract that and export that as a JAR

2. We can be backwards compatible for now but for a major release we should force people to only use libraries from JAR artifacts and not Wars

3. I updated the link (thanks for pointing out)

Will add some comments here

Thanks